src/DB_Command.php

```diff
@@ -904,6 +904,24 @@ public function tables( $args, $assoc_args ) {
 	 * [--all-tables]
 	 * : List all tables in the database, regardless of the prefix, and even if not registered on $wpdb. Overrides --all-tables-with-prefix.
 	 *
+	 * [--order=<order>]
+	 * : Ascending or Descending order.
+	 * ---
+	 * default: ASC
+	 * options:
+	 *   - ASC
+	 *   - DESC
```

Comment:
I think my preference is for `asc` and `desc`. `wp comment meta list` and `wp option list` both use lowercase ([ref](https://developer.wordpress.org/cli/commands/comment/meta/list/), [ref](https://developer.wordpress.org/cli/commands/option/list/)), so I think there's sufficient prior art there.

Reply:
When I searched for `--order` the first example that came up was form the profile command, which uses uppercase 😅 

I see now that the consensus is lowercase.

Reply:
Resolved by https://github.com/wp-cli/db-command/pull/226/commits/621ee4bc976698fd348ab510959557b8c7f73d2f